config/schema/contentpool_client.schema.yml

```diff
@@ -11,3 +11,13 @@ relaxed.remote.*.third_party.contentpool_client:
     remote_site_uuid:
       type: string
       label: 'The uuid of the contentpool site.'
+    channels:
```

Comment:
as it needs to be generic, I think best we add a "filter" array with keys being fields and values being field values to filter on.